requirements/requirements_ansible.in

```diff
@@ -30,14 +30,15 @@ azure-graphrbac==0.40.0
 # AWS
 boto==2.47.0    # last which does not break ec2 scripts
 boto3==1.6.2
-google-auth==1.6.2    # needed for gce inventory imports
+google-auth==1.6.2    # needed for gce inventory plugin imports, libcloud is git dependency
 jinja2==2.10    # required for native jinja2 types for inventory compat mode
 # netconf for network modules
 ncclient==0.6.3
 # netaddr filter
 netaddr
 # oVirt/RHV
 ovirt-engine-sdk-python==4.2.4    # minimum set inside Ansible facts module requirements
+pycurl==7.43.0.1    # higher versions will not install without SSL backend specified
```

Comment:
I need to return to this issue. Testing locally, outside of AWX, I see errors with `pip install pycurl==7.43.0.2` but no errors with `pip install pycurl==7.43.0.1`. The [release notes](http://pycurl.io/docs/latest/release-notes.html) do not help in assessing this problem. This issue seems to be the root cause:

https://github.com/pycurl/pycurl/issues/526

That has already been resolved, in that it was changed from an error to a warning.

However, if I install from recent source code, I still get an error, locally with Mac. Trying this again in the AWX venv with the development version of pycurl, I get the same error. They are both variants of:

> clang: error: no input files

> gcc: fatal error: no input files

Reply:
Filed https://github.com/oVirt/ovirt-engine-sdk/issues/18